system/modules/core/library/Contao/Folder.php

```diff
@@ -339,6 +340,7 @@ protected function getHash()
 	{
 		$arrFiles = array();
 
+		/** @var \RecursiveDirectoryIterator $it */
```

Comment:
Yep :)

There is another strange PHP behavior going on in the `Theme` class, which I however did not fix: "Method 'getAttribute' not found in class \DOMNode" :)